pay-api/src/pay_api/models/fee_schedule.py

```diff
@@ -159,6 +160,44 @@ def find_all(
 
         return query.all()
 
+    @classmethod
+    def get_fee_details(cls,product_code:str = None):
+        """Get detailed fee information including corp type, filing type, and fees."""
+         # Create aliases for the fee_codes table
+        main_fee_code = aliased(FeeCode)
+        service_fee_code = aliased(FeeCode)
+        
+         # Get the current system date
+        current_date = datetime.now(tz=timezone.utc).date()
+
+        query = db.session.query(
+            CorpType.code.label("corp_type"), 
+            FilingType.code.label("filing_type"),  
+            CorpType.description.label("product"),  
```

Comment:
Shouldn't be product and description be different?

Products are the table groupings that are in the UX and the description is used in the rows for the table (the "fees")

Reply:
I see you renamed it to description, that should be fine. Don't you still need the product code for grouping it on the UI?